src/main/resources/fabric.mod.json

```diff
@@ -3,8 +3,8 @@
   "id": "forcecloseworldloadingscreen",
   "version": "${version}",
 
-  "name": "kennytv's epic force close loading screen mod for Fabric",
```

Comment:
no

Reply:
The problem with the previous name is that then the client tag cannot be displayed as intended.

Reply:
Well it's not a bug in *this* mod, and see https://github.com/kennytv/kennytvs-epic-force-close-loading-screen-mod-for-fabric/pull/24

Reply:
Will cancel my PR then.